core/src/lib.rs

```diff
@@ -28,27 +28,19 @@ pub mod drop_bank_service;
 pub mod duplicate_repair_status;
 pub mod fetch_stage;
 pub mod fork_choice;
-pub mod forward_packet_batches_by_accounts;
```

Comment:
Unless I'm missing something obvious, I don't think all of these items needed to be public in the first place. Technically, we will limiting the scope of some of these items with this change, but don't think that is an issue.